specs/dwritecore/DWriteCore.md

```diff
@@ -1050,9 +1230,185 @@ int __cdecl wmain(int argc, WCHAR** argv)
 }
 ```
 
-# API Details
+### Rendering Color Glyph Runs
+
+In most cases, the simplest way to render glyph runs with color is to use either Direct2D's or
+DirectWrite's `DrawGlyphRunWithColor` method.
```

Comment:
Though I realize it's almost trivial, it would be good to have a short example of this as well.

Reply:
I've replaced this with a new sample that has two code paths. If the `ID2D1DeviceContext7` is available, it calls `DrawGlyphRunWithColor`. Otherwise, it calls `TranslateColorGlyphRun`.